client/src/components/SearchForm.tsx

```diff
@@ -54,12 +60,19 @@ function SearchForm() {
             </div>
           
           <div className="py-3">
-            <button className="mx-auto py-2 px-4 rounded-md flex items-center justify-center
-                             bg-cyan-600 hover:bg-cyan-700 transition duration-300
-                             text-white text-sm font-bold" 
-                    type="submit">
-              Go
-            </button>
+            <Link 
+                href={{
+					pathname: "/navigation",
+                	query: data 
+                }}
+            >
+				<button className="mx-auto py-2 px-4 rounded-md flex items-center justify-center
+								bg-cyan-600 hover:bg-cyan-700 transition duration-300
+								text-white text-sm font-bold" 
+						type="submit">
+				Go
+				</button>
+            </Link>
```

Comment:
Please just fix the indentation to improve readability. It's all over the place.